recipes/gpyreg/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gpyreg" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gpyreg-{{ version }}.tar.gz
+  sha256: c642081e35d1e7e149de1f031033f90db1a1cf21425c48c46c26694dea39b81d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.9
+    - matplotlib-base >=3.5.1
+    - numpy >=1.22.1
+    - pytest >=6.2.5
+    - pytest-rerunfailures >=10.2
+    - scipy >=1.7.3
+
+test:
+  imports:
+    - gpyreg
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/GPyReg/
+  doc_url: https://acerbilab.github.io/gpyreg/
+  dev_url: https://github.com/acerbilab/gpyreg/
```

Comment:
This URL is currently 404 for me. Could it be private?

Reply:
Sorry about that, the URL should work now (we just moved from private to public).